regression/cbmc/CVC4_Not_Present/test.desc

```diff
@@ -0,0 +1,8 @@
+CORE
+main.c
+--cvc4
+^SMT2 solver \'cvc4\' is not on the PATH$
+^EXIT=1$
+^SIGNAL=0$
+--
+^\[main.assertion.1\] line \d+ assertion i \> 5\: ERROR$
```

Comment:
We shouldn't include such a test as it would fail for someone who has `cvc4` on the `PATH`. Maybe `check_smt2` could be tested via a unit test instead.

Reply:
This test could be run with its own `chain.sh` that sets up an empty `PATH` before running cbmc.